Sources/Support/StoreMessagesHelper.swift

```diff
@@ -46,13 +46,15 @@ actor StoreMessagesHelper: StoreMessagesHelperType {
 
     #if os(iOS) || targetEnvironment(macCatalyst) || VISION_OS
 
-    func deferMessagesIfNeeded() async throws {
+    // `nonisolated` required to work around Swift 5.10 issue.
+    // See https://github.com/RevenueCat/purchases-ios/pull/3599
+    nonisolated func deferMessagesIfNeeded() async throws {
```

Comment:
More context for this: https://twitter.com/nachosoto/status/1750634899810472087?s=61&t=gxxxEpOJpZ2mMeKVPUPKew